.clang-format

```diff
@@ -0,0 +1,113 @@
+Language: Cpp
+
+AlignArrayOfStructures: Left
+AlignConsecutiveAssignments: AcrossComments
+AlignConsecutiveBitFields: None
+AlignConsecutiveDeclarations: None
+AlignConsecutiveMacros: AcrossComments
+AlignEscapedNewlines: Right
+AlignOperands: Align
+AlignTrailingComments: true
+
+AllowAllArgumentsOnNextLine: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: Empty
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortEnumsOnASingleLine: true
```

Comment:
```suggestion
AllowShortEnumsOnASingleLine: false
```
should do the trick

Reply:
I'll try that in a sec. It might be the case that clang-format needs these things to have trailing commas 🤔 I'll get back in a sec.

Reply:
Sorry, that was longer than a sec, but it works!